adafruit_tmp11X.py

```diff
@@ -539,4 +565,17 @@ def _read_status(self) -> Tuple[int, int, int]:
         return (high_alert, low_alert, data_ready)
 
     def _read_temperature(self) -> float:
-        return self._raw_temperature * _TMP117_RESOLUTION
+        return round(self._raw_temperature * _TMP117_RESOLUTION, 3)
```

Comment:
Don't round here, because the extra digits are still interesting when summing or averaging. Leave it up to the caller to round as needed.
```suggestion
        return self._raw_temperature * _TMP117_RESOLUTION```
```
Also this specifically mentions the TMP117 resolution: does the value change? Should it be `TMP11X_RESOLUTION`?